### PR TITLE
Leaderboard endpoints

### PR DESCRIPTION
**Pull request by GreenGard**

### CODE REVIEW

File: backend/entities/Leaderboard.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    @Column()
+    time: number;
+
+    @ManyToOne(() => Account, ((account) => account.leaderboard))
+    account: Account | undefined;
+
+    @ManyToOne(() => Account, ((account) => account.id))
+    accounts: Account | undefined;
+}
```
Comment (ToniKaru) [inline backend/entities/Leaderboard.ts:26]: Add new line at end of file.

File: backend/entities/Leaderboard.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    @Column()
+    score: number;
+
+    @Column()
+    time: number;
+
+    @ManyToOne(() => Account, ((account) => account.leaderboard))
+    account: Account | undefined;
+
```
Comment (DarkendHall) [inline backend/entities/Leaderboard.ts:23]: Feels a bit odd to have account linked twice to the same entity.

File: backend/entities/Account.ts [hunk 3]
```diff
@@ -23,6 +27,9 @@ export class Account {
     @Column()
     displayName: string;
 
+    @OneToMany(() => Leaderboard, (leaderboard) => leaderboard.account)
+    leaderboard: Leaderboard | undefined;
```
Comment (DarkendHall) [inline backend/entities/Account.ts:31]: Don't think it needs to be linked in both entities, should be enough to have ManyToOne in leaderboard entity.

Also there are some multiple empty lines and empty lines at the start of the file :)

File: backend/routes/leaderboard.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+            where: { id: userId },
+        })
+        .then((leaderboard) => {
+            res.status(200).json(leaderboard);
+        })
+        .catch((error) =>
+            res.status(404).json({ errorMessage: `${userId} doesn't exist`, error: error }),
+        );
+});
+
```
Comment (DarkendHall) [inline backend/routes/leaderboard.ts:27]: Do we really need both of these to get single user?

**Review by DarkendHall — CHANGES_REQUESTED** [review]
Other than the changes i requested it look good :), great job 👍

File: backend/routes/leaderboard.ts (hunk 4 above)
Comment (GreenGard) [reply backend/routes/leaderboard.ts:27]: Yes we might need them later

**Review by DarkendHall — APPROVED** [review]

**Review by ToniKaru — APPROVED** [review]
Great job!

**Pull request merged**